graph/resolvers/asset.js

```diff
@@ -1,4 +1,11 @@
 const Asset = {
+  lastComment({id}, _, {loaders: {Comments}}) {
+    return Comments.getByQuery({
+      asset_id: id,
+      limit: 1,
+      parent_id: null
+    }).then(data => data[0]);
```

Comment:
2. Oh yes we could, but semantically this is really nice on the client. Otherwise I'll get a data property `lastComment` and it's an array with 1 comment instead of just an object.